### DISCUSSION TITLE
tutorials/l2evpn/evpn/

### DISCUSSION THREAD

**Discussion by giscus** [srl-labs/learn-srlinux#55] [comments]
# tutorials/l2evpn/evpn/

Open documentation for open Network OS

https://learn.srlinux.dev/tutorials/l2evpn/evpn/

**Comment by thno42** [comment]
Great example, thanks. Now with srlinux 24 you need brackets ([]) around your policy names.

**Reply by hellt** [reply]
yeah, we pinned the lab example to SR Linux 23.10.1, now it is time to update it to 24.10.1

**Comment by manumamm** [comment]
all your diagrams are very nice (and very helpfull to understand), what tool do you use to generate/create them please?

**Reply by hellt** [reply]
the diagrams are done with draw.io
